source/features/network-info/store.ts

```diff
@@ -84,6 +85,7 @@ export class NetworkInfoStore extends Store {
     const result = await this.networkInfoApi.fetchStatic.execute({});
     if (result) {
       const { genesis } = result;
+      this.shelleyEpochLength = genesis.shelley?.epochLength || 21600
```

Comment:
Default should there be no Shelley genesis loaded in the API defaults to a familiar value